noxfile.py

```diff
@@ -118,10 +118,10 @@ def install_ci(session, group):
         )
         run(session, "uv pip install --system vitessce")
     elif group == "curator":
-        extras += "zarr,jupyter"
+        extras += "zarr,jupyter,bionty"
         run(
             session,
-            "uv pip install --system ./sub/bionty ./sub/wetlab",
+            "uv pip install --system ./sub/wetlab",
```

Comment:
The `./sub/bionty` thing has to be restored. If it causes an issue, it has to be fixed in bionty.

Reply:
I think it's the pinning of lamindb in bionty

Reply:
But bionty should only lower bound lamindb, not pin it 🤔 -- otherwise there will be issues

Reply:
This is compatible, everything should pass: https://github.com/laminlabs/bionty/blob/559cc9a301857aade2f9c739ce935f6e24121b5d/pyproject.toml#L19

If it doesn't that might be dangerous.